<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Product</class>
 <widget class="QDialog" name="Product">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>455</width>
    <height>275</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Product</string>
  </property>
  <property name="windowIcon">
   <iconset resource="smartbill.qrc">
    <normaloff>:/Icons/smartbill.jpeg</normaloff>:/Icons/smartbill.jpeg</iconset>
  </property>
  <widget class="QLabel" name="productNameLabel">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>12</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Product Name*</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="productNameLineEdit">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>33</y>
     <width>431</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="supplierNameLabel">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>62</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Supplier Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="priceLabel">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>60</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Price*</string>
   </property>
  </widget>
  <widget class="QLabel" name="quanitityLabel">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>60</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Quantity*</string>
   </property>
  </widget>
  <widget class="QLabel" name="productDescpnLabel">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>113</y>
     <width>119</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Product Description</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="supplierNameLineEdit">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>83</y>
     <width>241</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="quanititySpinBox">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>83</y>
     <width>71</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100000000</number>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="priceDoubleSpinBox">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>83</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximum">
    <double>1000000000.000000000000000</double>
   </property>
  </widget>
  <widget class="QPushButton" name="addProductInPushButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>240</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelAddProductPushButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>240</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="productDescpnLineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>431</width>
     <height>101</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <zorder>productDescpnLineEdit</zorder>
  <zorder>productNameLabel</zorder>
  <zorder>productNameLineEdit</zorder>
  <zorder>supplierNameLabel</zorder>
  <zorder>priceLabel</zorder>
  <zorder>quanitityLabel</zorder>
  <zorder>productDescpnLabel</zorder>
  <zorder>supplierNameLineEdit</zorder>
  <zorder>quanititySpinBox</zorder>
  <zorder>priceDoubleSpinBox</zorder>
  <zorder>addProductInPushButton</zorder>
  <zorder>cancelAddProductPushButton</zorder>
 </widget>
 <tabstops>
  <tabstop>productNameLineEdit</tabstop>
  <tabstop>supplierNameLineEdit</tabstop>
  <tabstop>priceDoubleSpinBox</tabstop>
  <tabstop>quanititySpinBox</tabstop>
  <tabstop>productDescpnLineEdit</tabstop>
  <tabstop>addProductInPushButton</tabstop>
  <tabstop>cancelAddProductPushButton</tabstop>
 </tabstops>
 <resources>
  <include location="smartbill.qrc"/>
 </resources>
 <connections/>
</ui>
